nix/machines/_common/base.nix

```diff
@@ -36,15 +36,44 @@
     tmux
     usbutils
     unixtools.nettools
-    vim
     wget
+    ((vim_configurable.override { }).customize {
+      name = "vim";
+      # Install plugins for example for syntax highlighting of nix files
+      vimrcConfig.packages.myplugins = with pkgs.vimPlugins; {
+        start = [
+          vim-nix
+          vim-lastplace
```

Comment:
lets get rid of `vim-lastplace` I dont think we'd have much use for it on the server